src/content/docs/custom-pipeline/setup.mdx

```diff
@@ -1,209 +1,186 @@
 ---
 title: Setting up your computer
-description: Setting up your computer for creating a custom pipeline.
+description: Setting up your computer to create a custom pipeline.
 ---
 
-## Prototype pipeline development container
+import CloneIcon from '~icons/codicon/source-control'
+import RemoteIcon from '~icons/codicon/remote'
+import ExtensionsIcon from '~icons/codicon/extensions'
+import CoffeeIcon from '~icons/codicon/coffee'
+import WaitIcon from '~icons/ic/round-access-time'
 
-## Requirements
+## Quick setup : VSCode and **devcontainers**
 
-- [VS Code](https://code.visualstudio.com) &geq; 1.95
-- [Docker](https://www.docker.com/get-started/) &geq; 24 (we recommend using [Docker Desktop](https://www.docker.com/products/docker-desktop))
+### Prerequisites
```

Comment:
I would add before a small paragraph of the purpose of this section. "At the end of this setup you will have created an environment with all the tools needed to start creating your nf-neuro pipeline." Something like this but better :D